server/src/logic/handler/routes/recipe.handler.ts

```diff
@@ -31,4 +88,45 @@ export class RecipeHandler {
       })
       .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
   }
+
+  public update = (req: Request, res: Response): void => {
+    const newRecipe: Recipe = req.body as Recipe;
+
+    this._db.users.getUserById(newRecipe.userId)
+      .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
+
+    this._db.recipes.update(req.params.id, newRecipe)
+      .then(() => {
+        this.getById(req, res);
+      })
+      .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
+  }
+
+  public updateRecipe = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
i think this should be moved into a middleware. maybe into the ModelValidationMiddleware class.